src/main/java/net/minestom/server/instance/InstanceContainer.java

```diff
@@ -123,6 +124,85 @@ public void setSeparateBlocks(int x, int y, int z, short blockStateId, short cus
         setBlock(x, y, z, blockStateId, customBlock, data);
     }
 
+    @Override
+    public void placeBlock(@NotNull Player player, @NotNull Chunk chunk, @NotNull BlockFace blockFace, @NotNull BlockPosition blockPosition, short blockId, short customBlockId, @Nullable Data data) {
+        setBlock(player, chunk, blockFace, blockPosition, blockId, customBlockId, data);
+    }
+
+    @Override
+    public void setBlock(@NotNull BlockPosition blockPosition, short blockId, short customBlockId, @Nullable Data data) {
```

Comment:
None I suppose, other than the fact that it uses another method to actually place the blocks. I kept `setSeparateBlocks` because I didn't want to remove the old way.